indico/modules/events/layout/forms.py

```diff
@@ -85,6 +88,8 @@ def validate_use_custom_css(self, field):
 
 
 class LectureMeetingLayoutForm(IndicoForm):
+    name_format = IndicoEnumSelectField(_('Name format'), enum=NameFormat, default=NameFormat.inherit,
+                                        description=_('Default format in which names are displayed'))
```

Comment:
This is not a "default" since it forces the selected format regardless of the user's preference.